client/app/components/groupInfo/groupInfo.html

```diff
@@ -2,7 +2,7 @@
   <md-toolbar class="md-hue-1">
     <div class="md-toolbar-tools">
       <h1 class="md-title" flex>
-        <md-button ui-sref="groupList" class="md-icon-button">
+        <md-button ui-sref="joinGroupList" class="md-icon-button">
```

Comment:
The test setup for e2e is still a bit of manual work, I'm afraid. I think you can avoid setting up the backend if you run the e2e-tests against dev.foodsaving.world for now.